### ISSUE TITLE
MysqlAdapter

### ISSUE THREAD

**Issue by victorgp89** [ethanresnick/json-api#178]
It's posible an integration with mysql stack?

**Comment by ethanresnick** [comment]
There isn't an adapter off-hand, but @carlbennettnz has been working on a Postgres adapter, which could probably be used with MySQL either directly or with very limited modifications. I believe that's here: https://github.com/carlbennettnz/json-api-knex-adapter If you need help modifying it for your purposes, I'm happy to answer questions, although the MongooseAdapter hopefully provides a pretty good example of the required methods and what each one does.

**Comment by victorgp89** [comment]
Have you a version of this api without typescript?

**Comment by ethanresnick** [comment]
@victorgp89 No, but you can easily compile the Typescript (e.g., by pasting it [here](http://www.typescriptlang.org/play/index.html)), tweak the output a bit, and then you'll have idiomatic JS without too much work.

**Comment by tesserakt** [comment]
Piggybacking on this, have you thought of integration with something like https://github.com/typeorm/typeorm ? Perhaps their datamapper approach?
